hieradata_aws/class/integration/account_api_db_admin.yaml

```diff
@@ -0,0 +1,23 @@
+govuk_env_sync::tasks:
+  "pull_account_api_production_daily":
+    ensure: "present"
+    hour: "0"
+    minute: "0"
+    action: "pull"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "account-api_production"
+    temppath: "/tmp/account_api_production"
+    url: "govuk-production-database-backups"
+    path: "account-api-postgresql"
+  "push_account_api_production_daily":
+    ensure: "present"
+    hour: "5"
+    minute: "0"
+    action: "push"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "account-api_production"
+    temppath: "/tmp/account_api_production"
+    url: "govuk-integration-database-backups"
+    path: "account-api-postgresql"
```

Comment:
Let's not push in the first iteration as otherwise we'll have both the main db-admin and this trying to write the same file.